code/game/machinery/Sleeper.dm

```diff
@@ -17,12 +17,12 @@
 	var/list/available_chems
 	var/controls_inside = FALSE
 	var/list/possible_chems = list(
-		list("epinephrine", "morphine", "salbutamol", "bicaridine", "kelotane"),
+		list("epinephrine", "morphine", "salbutamol", "styptic_powder", "silver_sulfadiazine"),

```

Comment:
i dont think you want to inject those

Reply:
they do work

Reply:
Styptic powder and silver sulfadiazine currently cause an instant tox loss equal to half the volume on reaction_mob() if the method is VAPOR, INJECT or INGEST.
So yea, good luck unless you remove its tox loss quirk.

Reply:
This really makes me think